dataflux_pytorch/benchmark/checkpointing/multinode/README.md

```diff
@@ -4,56 +4,115 @@ This benchmarking script will allow you to run and benchmark the performance of
 
 ## Getting started
 
-### Configuration
+### Setup
+ 
+#### Install requirements
+Create a Python virtual environment and activate it
+```shell
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+Run the following commands from the root of the repo to install the packages needed by the benchmarking code
+
+```shell
+pip install -r dataflux_pytorch/benchmark/requirements.txt`
+pip install .
+```
+
+#### Auth
 
-First ensure you are running within a virtual python enviroment, make sure gcloud config project is set to correct value. Otherwise use the following command to set it 
+Set up credentials for the benchmarking code to be able to access your GCS bucket
```

Comment:
"Set up credentials for the benchmark code to access your GCS bucket."

Reply:
Done.